content_schemas/dist/DO-NOT-EDIT-FILES-IN-HERE.md

```diff
@@ -0,0 +1,6 @@
+# Do not edit files in `dist/`
+
+This folder contains generated files. You should not directly edit these
+files. They will be regenerated by running `rake`.
+
+For more info see the [README.md](/README.md).
```

Comment:
```suggestion
For more info see the [README.md](../README.md).
```
This currently links to the main README for Publishing API, rather than the README for content schemas.